extensionsGallery-insider.json

```diff
@@ -553,7 +553,7 @@
 				},
 				{
 					"extensionId": "19",
-					"extensionName": "sqldw-insights",
+					"extensionName": "sql-dw",
```

Comment:
is this necessary? a potential impact to the users who already installed the extension

Reply:
I don't know about "necessary" - but right now the package.json has `sql-dw` so they should match to be accurate.

What impact do you see? The extension name in the gallery shouldn't impact anything for the installed extension (it's always been sql-dw).